src/components/dropdown/dropdown.tsx

```diff
@@ -199,55 +188,36 @@ export const Dropdown: FC<DropdownProps> = ({
     ));
 
   return (
-    <Manager>
+    <div ref={containerRef} className={cx('container', className)} title={title}>
+      <button
+        {...getToggleButtonProps({
+          tabIndex: disabled ? -1 : 0,
```

Comment:
do we need that considering that the element is not focused by default if it's disabled?

Reply:
Good catch, thanks!
Actually, no.
Can be replaced by simply providing the `disabled` prop, since the `toggleButton` is now represented as a `button` element.